ci-operator/step-registry/openshift/e2e/nutanix/ovn/openshift-e2e-nutanix-ovn-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: openshift-e2e-nutanix-ovn
+  steps:
+    pre:
+    - chain: ipi-conf-nutanix
+    - ref: ovn-conf
```

Comment:
this is only needed if you have nutanix-ovn jobs that are running on 4.11 or before. In 4.12 and above, OVN will be there by default. This doesn't hurt, per se, as it will explicitly configure OVN to be there. essentially a noop I guess.